style-desktop.css

```diff
@@ -0,0 +1,316 @@
+    font-size: 48px;
+    font-weight: 800;
+    line-height: 60px;
+  }
+
+  .paragraphe {
+    text-align: center;
+    justify-content: center;
+    font-size: 20px;
+    width: 920px;
+    line-height: 32px;
+    padding: 0;
+    margin: 0 954px;
+  }
+
+  .paragraphe p {
+    text-align: center;
+    justify-content: center;
+    padding: 12px;
+    margin: 24px;
+  }
+
+  .social--icons li {
+    padding-top: 58px;
+    margin-right: 20px;
+    width: 19px;
+    height: 20px;
+  }
+
+  .main--works--section {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 24px;
+    grid-auto-rows: 474px;
+    height: auto;
+    text-align: center;
+    padding: 0 750px;
+    padding-top: 24px;
+    margin-top: 90px;
+  }
+
+  .main--frames--section {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 24px;
+    grid-auto-rows: 470px;
+    text-align: center;
+    margin: 0;
+    padding: 20px 750px;
+    padding-top: 24px;
+  }
+
+  .card--section {
+    margin-right: 100px;
+    width: 100%;
+    height: 100%;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .intro--section {
+    text-align: center;
+    justify-content: center;
+    padding-bottom: 63px;
+    margin: 0;
+    width: 100%;
+  }
+
+  .about-title {
+    margin: 0;
+    padding-top: 192px;
+  }
+
+  .recent--work--section,
+  .about--me--section {
+    padding: 0;
+    margin: 0;
+  }
+
+  .about--me--section {
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    justify-content: center;
+    background: none;
+    background-image:
+      url(./assets/back-img/illustration-left-about-me-2-desktop@2x.png),
+      url(./assets/back-img/illustration-right-about\ me-1-desktop@2x.png);
+    background-position: left 27%, right 3%;
+    background-size: 11%;
+    background-repeat: no-repeat;
+  }
+
+  .frame--work--section {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    border: 1px solid #d0d9d4;
+    background: #ebf0ee;
+    border-radius: 8px;
+  }
+
+  .contact--intro {
+    margin-right: 20px;
+    padding: 0;
+  }
+
+  .contact--intro h1 {
+    padding: 0;
+    margin-top: 125px;
+    color: #172b4d;
+    font-size: 30px;
+    line-height: 52px;
+    font-weight: bold;
+    width: 610px;
+    height: auto;
+  }
+
+  .contact--section {
+    background: none;
+    display: flex;
+    margin: 70px 0;
+    padding: 64px 24px;
+    justify-content: center;
+    text-align: center;
+    background-image:
+      url(./assets/back-img/illustration-1-contact-form-desktop@2x.png),
+      url(./assets/back-img/illustration-2-contact-form-desktop@2x.png),
+      url(./assets/back-img/contact\ form-illustratoin-3-desktop@2x.png);
+    background-position: left 90%, 20% bottom, right bottom;
+    background-size: 8%;
+    background-repeat: no-repeat;
+    padding-bottom: 40px;
+  }
+
```

Comment:
- [ ] Kindly implement the recommended changes for this section also. 

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144666123-e6f25740-7b59-4cc8-895a-a3bc4faa1bb2.png)

## Figma Template

![image](https://user-images.githubusercontent.com/77106962/144666265-aa4b7f80-05a2-48ef-a8a1-359f4cdb9777.png)

### Kindly implement the changes 🙏

Reply:
Hey Gambit, thanks for the review you gave me, I actually this section was said to be ignored in the requirements .they said we should only use full name input tag for all versions and it is what I have done so far, would you please say something so the next review will not consider this section?